<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefDialog</class>
 <widget class="QDialog" name="PrefDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>141</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::SinglePass</enum>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="gridSize">
      <size>
       <width>130</width>
       <height>82</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
     <property name="uniformItemSizes">
      <bool>false</bool>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Open options</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/document-import.png</normalon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Copy options</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/download.png</normalon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Resize options</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/transform-scale.png</normalon>
       </iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>CSV delimiter : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_delimiter"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="chFailed">
         <property name="text">
          <string>Open only failed images</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QTreeView" name="listFields"/>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Copy mode</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="chLocal">
            <property name="text">
             <string>Copy local</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="chFtp">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Copy to FTP (needed enter host, password, etc.)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="chBridges">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Copy to target use bridge (in DEVELOPMET STATE)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Files copy options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="rSkipCopy">
            <property name="toolTip">
             <string>Skip copy if file is exist</string>
            </property>
            <property name="text">
             <string>Skip</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rReplace">
            <property name="toolTip">
             <string>replace if file is exist</string>
            </property>
            <property name="text">
             <string>Replace</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_presta">
         <property name="toolTip">
          <string>Automate convert file to JPG format</string>
         </property>
         <property name="text">
          <string>Automate convert to JPG</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Resize methods:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0" colspan="2">
             <widget class="QRadioButton" name="prestaMethod">
              <property name="text">
               <string>PrestaShop resize method</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="numPrestaMethod">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>2</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="normalMethod">
            <property name="text">
             <string>Normal resize method</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QPushButton" name="okBtn">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;Copyright (C) 2010 Peter Komar. Development for cart2cart service&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okBtn</sender>
   <signal>clicked()</signal>
   <receiver>PrefDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prestaMethod</sender>
   <signal>toggled(bool)</signal>
   <receiver>numPrestaMethod</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
